src/pages/Dashboard/Apps/Add/Card/index.scss

```diff
@@ -1,5 +1,19 @@
 @import '~scss/vars';
 
+@mixin disable-hover($opacity) {
```

Comment:
Since this mixin is only used once, i think no need to define it. Also the children css module will prevent reuse